basics/storage-classes.md

```diff
@@ -0,0 +1,86 @@
+# Speicherklassen
+
+D besitzt eine statische Typisierung. Wenn eine Variable deklariert wurde, kann der
+Typ nicht mehr verändert werden. Dies erlaubt dem Kompiler viele Fehler früh
+zu erkennen und Restriktionen können schon während der Kompilierung erkannt werden.
+Ein gutes Typsystem leistet die benötigte Unterstützung um große Programme
+sicher und leichter zu warten zu machen.
+
+### `immutable`
+
+Zusätzlich zu der statischen Typisierung, stellt D Speicherklassen bereit
+welche weitere Beschränkungen erlauben. Zum Beispiel kann ein `immutable` (unveränderbares)
+Objekt nur einmal initialisiert werden und kann danach nicht mehr verändert werden.
+
+    immutable int err = 5;
+    // oder: immutable arr = 5 (der Typ int wird automatisch inferriert)
+    err = 5; // FEHLER
+
+Wegen ihrer Unveränderbarkeit, können `immutable` Objekte sicher zwischen
+Threads geteilt werden. Dies impliziert auch das `immutable` Objekte perfekt
+gecacht werden können.
+
+### `const`
+
+`const` Objekte können ebenfalls nicht verändert werden. Diese Restriktion ist
+jedoch nur für den aktuellen Geltungsbereich gültig. Ein `const` Zeiger
+kann von sowohl einem `immutable` als auch _veränderbaren_ Objekt erzeugt werden.
```

Comment:
`sowohl von, ... als auch von einem` instead of `von sowohl, ... als auch` is easier to read imo